Windows plugin build is broken:
```
error[E0412]: cannot find type `Device` in this scope
   --> ports\gstplugin\servowebsrc.rs:103:17
    |
103 | type Context = <Device as DeviceAPI>::Context;
    |                 ^^^^^^ not found in this scope
    |
help: possible candidates are found in other modules, you can import them into scope
    |
5   | use gstreamer::Device;
    |
5   | use surfman::Device;
    |
5   | use surfman::device::Device;
    |
5   | use surfman::platform::default::device::Device;
    |
      and 1 other candidate

error[E0412]: cannot find type `Device` in this scope
   --> ports\gstplugin\servowebsrc.rs:104:20
    |
104 | type Connection = <Device as DeviceAPI>::Connection;
    |                    ^^^^^^ not found in this scope
    |
help: possible candidates are found in other modules, you can import them into scope
    |
5   | use gstreamer::Device;
    |
5   | use surfman::Device;
    |
5   | use surfman::device::Device;
    |
5   | use surfman::platform::default::device::Device;
    |
      and 1 other candidate

error[E0412]: cannot find type `Device` in this scope
   --> ports\gstplugin\servowebsrc.rs:105:23
    |
105 | type NativeContext = <Device as DeviceAPI>::NativeContext;
    |                       ^^^^^^ not found in this scope
    |
help: possible candidates are found in other modules, you can import them into scope
    |
5   | use gstreamer::Device;
    |
5   | use surfman::Device;
    |
5   | use surfman::device::Device;
    |
5   | use surfman::platform::default::device::Device;
    |
      and 1 other candidate

error[E0412]: cannot find type `Device` in this scope
   --> ports\gstplugin\servowebsrc.rs:138:13
    |
138 |     device: Device,
    |             ^^^^^^ not found in this scope
    |
help: possible candidates are found in other modules, you can import them into scope
    |
5   | use gstreamer::Device;
    |
5   | use surfman::Device;
    |
5   | use surfman::device::Device;
    |
5   | use surfman::platform::default::device::Device;
    |
      and 1 other candidate

error[E0412]: cannot find type `Device` in this scope
   --> ports\gstplugin\servowebsrc.rs:140:27
    |
140 |     swap_chain: SwapChain<Device>,
    |                           ^^^^^^ not found in this scope
    |
help: possible candidates are found in other modules, you can import them into scope
    |
5   | use gstreamer::Device;
    |
5   | use surfman::Device;
    |
5   | use surfman::device::Device;
    |
5   | use surfman::platform::default::device::Device;
    |
      and 1 other candidate

error[E0412]: cannot find type `Device` in this scope
   --> ports\gstplugin\servowebsrc.rs:168:35
    |
168 |     GetSwapChain(Sender<SwapChain<Device>>),
    |                                   ^^^^^^ not found in this scope
    |
help: possible candidates are found in other modules, you can import them into scope
    |
5   | use gstreamer::Device;
    |
5   | use surfman::Device;
    |
5   | use surfman::device::Device;
    |
5   | use surfman::platform::default::device::Device;
    |
      and 1 other candidate

error[E0412]: cannot find type `Device` in this scope
   --> ports\gstplugin\servowebsrc.rs:182:27
    |
182 |     swap_chain: SwapChain<Device>,
    |                           ^^^^^^ not found in this scope
    |
help: possible candidates are found in other modules, you can import them into scope
    |
5   | use gstreamer::Device;
    |
5   | use surfman::Device;
    |
5   | use surfman::device::Device;
    |
5   | use surfman::platform::default::device::Device;
    |
      and 1 other candidate

error[E0412]: cannot find type `Device` in this scope
   --> ports\gstplugin\servowebsrc.rs:188:13
    |
188 |     device: Device,
    |             ^^^^^^ not found in this scope
    |
help: possible candidates are found in other modules, you can import them into scope
    |
5   | use gstreamer::Device;
    |
5   | use surfman::Device;
    |
5   | use surfman::device::Device;
    |
5   | use surfman::platform::default::device::Device;
    |
      and 1 other candidate

error[E0412]: cannot find type `Device` in this scope
   --> ports\gstplugin\servowebsrc.rs:292:27
    |
292 |     swap_chain: SwapChain<Device>,
    |                           ^^^^^^ not found in this scope
    |
help: possible candidates are found in other modules, you can import them into scope
    |
5   | use gstreamer::Device;
    |
5   | use surfman::Device;
    |
5   | use surfman::device::Device;
    |
5   | use surfman::platform::default::device::Device;
    |
      and 1 other candidate

error[E0282]: type annotations needed for `std::sync::Mutex<std::option::Option<T>>`
   --> ports\gstplugin\servowebsrc.rs:497:37
    |
497 |         let connection = Mutex::new(None);
    |             ----------              ^^^^ cannot infer type for `T`
    |             |
    |             consider giving `connection` the explicit type `std::sync::Mutex<std::option::Option<T>>`, where the type parameter `T` is specified

error: unused import: `surfman_chains_api::SwapChainAPI`
  --> ports\gstplugin\servowebsrc.rs:95:5
   |
95 | use surfman_chains_api::SwapChainAPI;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: `-D unused-imports` implied by `-D warnings`

error: aborting due to 11 previous errors
```

--- a/etc/taskcluster/decision_task.py
+++ b/etc/taskcluster/decision_task.py
@@ -491,9 +491,9 @@ def windows_unit(cached=True):
             "mach smoketest --angle",
             "mach package --dev",
             "mach build --dev --libsimpleservo",
-            # We're getting link errors on windows, due to the x11 feature being
-            # enabled on gstreamer-gl. https://github.com/servo/media/pull/304/
-            "mach build --dev --media-stack=dummy -p servo-gst-plugin",
+            # The GStreamer plugin currently doesn't support Windows
+            # https://github.com/servo/servo/issues/25353
+            # "mach build --dev -p servo-gst-plugin",
 
         )
         .with_artifacts("repo/target/debug/msi/Servo.exe",

